admin_app/src/app/content/page.tsx

```diff
@@ -2,70 +2,136 @@
 import type { Content } from "@/app/content/edit/page";
 import ContentCard from "@/components/ContentCard";
 import { Layout } from "@/components/Layout";
-import { LANGUAGE_OPTIONS, sizes } from "@/utils";
+import { LANGUAGE_OPTIONS, appColors, sizes } from "@/utils";
 import { apiCalls } from "@/utils/api";
 import { useAuth } from "@/utils/auth";
-import { Add } from "@mui/icons-material";
-import { Button, CircularProgress, Grid } from "@mui/material";
+import { Add, Sort } from "@mui/icons-material";
+import { Button, CircularProgress, FormControl, Grid, InputLabel, MenuItem, Select, useMediaQuery } from "@mui/material";
 import Alert from "@mui/material/Alert";
 import Snackbar from "@mui/material/Snackbar";
 import Link from "next/link";
 import { useSearchParams } from "next/navigation";
 import React from "react";
 import { PageNavigation } from "../../components/PageNavigation";
 import { SearchBar } from "../../components/SearchBar";
+import theme from "@/theme";
 
 const MAX_CARDS_PER_PAGE = 12;
 
+interface ContentLanding extends Content {
+  languages: string[];
+}
+interface Language {
```

Comment:
Added this interface temporarily because I didnt want to break the LanguageButtonBar. Will remove it in next PR